Numbers/unit_converter.py

```diff
@@ -0,0 +1,28 @@
+# 11 c -> f, 12 f -> c
+# 21EUR, 22UAH, 23USD ----> RUB
+# 31kg -> pound, 32pound -> kg
+
+def unit_converter(convert: int, value: int) -> str:
```

Comment:
change convert int to something else

Reply:
Rename "convert" to "direction".

Reply:
change signature of the method to
convert_unit(from, to, value)

"from" and "to" might be a list of strings like: 
celsius, fahrenheit, USD, RUB, etc.